### PR TITLE
Interaction fixes

### PR DESCRIPTION
**Pull request by Juniverse**

### CODE REVIEW

File: Runtime/Scripts/Interaction/InworldInteraction.cs [hunk 1]
```diff
@@ -16,7 +16,6 @@ namespace Inworld.Interactions
     public class InworldInteraction : MonoBehaviour
     {
         [SerializeField] GameObject m_ContinueButton;
-        [SerializeField] protected bool m_Interruptable = true;
```
Comment (mattkwilson) [inline Runtime/Scripts/Interaction/InworldInteraction.cs:19]: We should keep the global interruptable flag.

**Review by mattkwilson — APPROVED** [review]

File: Runtime/Scripts/PlayerControl/PlayerController.cs [hunk 2]
```diff
@@ -21,59 +21,26 @@ namespace Inworld.Sample
...
     {
-        [Header("References")]
-        [SerializeField] protected TMP_InputField m_InputField;
-        [SerializeField] protected TMP_Dropdown m_Dropdown;
-        [SerializeField] protected Button m_SendButton;
-        [SerializeField] protected Button m_RecordButton;
-
-        protected InputAction m_SubmitInputAction;
-        
+        //protected InputAction m_SubmitInputAction;
```
Comment (mattkwilson) [inline Runtime/Scripts/PlayerControl/PlayerController.cs:24]: Please remove

File: Runtime/Scripts/Audio/AudioCapture.cs [hunk 3]
```diff
@@ -383,56 +393,61 @@ protected void Update()
...
+            m_PrevSampleMode = m_SamplingMode;
+            m_SamplingMode = MicSampleMode.PUSH_TO_TALK;
+            m_PrevDetectPlayerSpeaking = m_DetectPlayerSpeaking;
+            m_DetectPlayerSpeaking = false;
+        }
+        protected virtual void OnPlayerCanvasClosed()
+        {
+            m_SamplingMode = m_PrevSampleMode;
+            m_DetectPlayerSpeaking = m_PrevDetectPlayerSpeaking;
+        }
```
[the remark this answers is not in the document]
Comment (Juniverse) [reply Runtime/Scripts/Audio/AudioCapture.cs:407]: Let's refactor later.

File: Runtime/Scripts/Interaction/InworldInteraction.cs (hunk 1 above)
Comment (Juniverse) [reply Runtime/Scripts/Interaction/InworldInteraction.cs:19]: [Let's make it there for n](https://inworldai.atlassian.net/browse/INTG-2091)

File: Runtime/Scripts/PlayerControl/PlayerController.cs (hunk 2 above)
Comment (Juniverse) [reply Runtime/Scripts/PlayerControl/PlayerController.cs:24]: https://inworldai.atlassian.net/browse/INTG-2092

**Pull request merged**